examples/xgboost/utils/baseline_centralized.py

```diff
@@ -20,84 +21,99 @@
 from sklearn.metrics import roc_auc_score
 from torch.utils.tensorboard import SummaryWriter
 
-# Specify training params
-data_path = "../dataset/HIGGS_UCI.csv"
-model_path_root = "../workspaces/centralized"
-round_num = 100
+parser = argparse.ArgumentParser(description="Centralized XGBoost training with random forest options")
+parser.add_argument("--num_parallel_tree", type=int, default=1, help="num_parallel_tree for random forest setting")
+parser.add_argument("--subsample", type=float, default=1, help="subsample for random forest setting")
```

Comment:
suggest put parser code inside main function.

Reply:
Changed as @chesterxgchen suggested, also changed others in similar manner